examples/17.initialise_readonly.ts

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Please merge this example with 1.initialisation example
in that show both ways of creating a client i.e. write mode and read mode